src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private int radius;
+
+    public Circle(String color, int radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: Circle");
+        System.out.println("Color: " + getColor());
+        System.out.println("Radius: " + radius);
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
```

Comment:
All magic numbers in your code should be constants. Consider defining a constant for `Math.PI` to improve code readability and maintainability.